### DISCUSSION TITLE
Integrate Wiki.js into my React app

### DISCUSSION THREAD

**Discussion by CodeByChaim** [requarks/wiki#5828] [Help / Questions] [answered]
Dear friend, 

I am building a React/Nginx app running as a container on ubuntu.
So far, I was able to deploy Wiki.js as a standalone app.
I would like to integrate Wiki.js into my app as a knowledgebase, and I would appreciate any assistant in this matter.
Additionally, I would like to have the ability to send search-terms into the wiki search from other section of the app.

Kind regards
Chaim
![wiki](https://user-images.githubusercontent.com/36452879/200605117-d3d28389-8c37-433d-bedc-4e850118171c.PNG)

**Comment by NGPixel** [comment, marked as answer]
Use an iframe.

Wiki.js is using Vue and has its own backend, so its not something can you easily integrate into another app.

**Comment by CodeByChaim** [comment]
Thanks for the quick reply 🙂
________________________________
